cosmos/operators/local.py

```diff
@@ -816,3 +816,12 @@ def __init__(self, **kwargs: str) -> None:
         raise DeprecationWarning(
             "The DbtDepsOperator has been deprecated. " "Please use the `install_deps` flag in dbt_args instead."
         )
+
+
+class DbtSourceLocalOperator(DbtLocalBaseOperator):
```

Comment:
There was a recent refactor, which exposes the dbt command as generic Mixin classes, in the `cosmos/operators/base.py` module:
* https://github.com/astronomer/astronomer-cosmos/blob/e61f3a365f6c3a0b550cda298516f64ad1321189/cosmos/operators/base.py#L347
* https://github.com/astronomer/astronomer-cosmos/blob/e61f3a365f6c3a0b550cda298516f64ad1321189/cosmos/operators/base.py#L379

And then we're able to simply subclass them in all modules of Cosmos operators, such as:
* https://github.com/astronomer/astronomer-cosmos/blob/main/cosmos/operators/local.py#L528
* https://github.com/astronomer/astronomer-cosmos/blob/main/cosmos/operators/kubernetes.py#L128

This was an improvement by @jbandoro which made the code much cleaner and testable. It would be great if we could follow the same pattern here, by:
* creating a `DbtSourceMixin` class in `operators/base.py`
* and inheriting it in all supported execution modes within the modules in `cosmos/operators`

Reply:
Done ✔️